SkietbaanBE/SkietbaanBE/Controllers/CompetitionController.cs

```diff
@@ -28,12 +27,19 @@ public IEnumerable<Competition> Get()
             var competitionIDsQuery = from Comp in _context.Competitions
                                       where Comp.Status == true
                                       select Comp;
-            List<Competition> competitionsList = competitionIDsQuery.ToList<Competition>();
-            // return _context.Competitions.ToArray<Competition>();
+            List<Competition> competitionsList = competitionIDsQuery.ToList<Competition>();     
             return competitionsList;
         }
-        //Getting the competition by ID
-        // GET: api/Competition/5
+        /** The method that return an array of competition objects whether status is true or false*/
+        // GET: api/Competition
+        [HttpGet("all")]
```

Comment:
This endpoint and function name needs to be more descriptive. I would suggest adding `competition/all`
and `GetALLCompetitions`